x-pack/plugins/cloud_defend/public/components/control_general_view_selector/index.tsx

```diff
@@ -400,7 +411,7 @@ export const ControlGeneralViewSelector = ({
                   <b>{i18n.conditions}</b>
                 </EuiText>
                 <EuiBadge
-                  title={conditionsAdded.join(',')}
+                  title={conditionsAdded.join(', ')}
```

Comment:
This is miscellaneous change aimed at improving readability in a tooltip (names we mashed together before):

![image](https://github.com/elastic/kibana/assets/25308326/c4c6f792-88fd-43a3-993b-f74fa9dea4a7)